src/tests/ftest/util/macsio_util.py

```diff
@@ -0,0 +1,486 @@
+        #       files from each processor for the main dump object prior to
+        #       starting dumps.
+        self.debug_level = FormattedParameter("--debug_level {}")
+
+        #
+        # Log File Options to control size and shape of log file:
+        #
+
+        #   --log_file_name %s [macsio-log.log]
+        #       The name of the log file.
+        self.log_file_name = FormattedParameter(
+            "--log_file_name {}", "macsio-log.log")
+
+        #   --log_line_cnt %d %d [64 0]
+        #       Set number of lines per rank in the log file and number of extra
+        #       lines for rank 0.
+        self.log_line_cnt = FormattedParameter("--log_line_cnt {}")
+
+        #   --log_line_length %d [128]
+        #       Set log file line length.
+        self.log_line_length = FormattedParameter("--log_line_length {}")
+
+        #   --timings_file_name %s [macsio-timings.log]
+        #       Specify the name of the timings file. Passing an empty string,
+        #       "" will disable the creation of a timings file.
+        self.timings_file_name = FormattedParameter(
+            "--timings_file_name {}", "macsio-timings.log")
+
+        #
+        # Options specific to the "hdf5" I/O plugin
+        #
+
+        #   --show_errors []
+        #       Show low-level HDF5 errors
+        self.show_errors = FormattedParameter("--show_errors {}")
+
+        #   --compression %s %s []
+        #       The first string argument is the compression algorithm name. The
+        #       second string argument is a comma-separated set of params of the
+        #       form 'param1=val1,param2=val2,param3=val3. The various algorithm
+        #       names and their parameter meanings are described below. Note
+        #       that some parameters are not specific to any algorithm. Those
+        #       are described first followed by individual algorithm-specific
+        #       parameters for those algorithms available in the current build.
+        #
+        #           minsize=%d [1024]
+        #               minimum size of dataset (in terms of a count of values)
+        #               upon which compression will even be attempted
+        #
+        #           shuffle=<int>
+        #               Boolean (zero or non-zero) to indicate whether to use
+        #               HDF5's byte shuffling filter *prior* to compression.
+        #               Default depends on algorithm. By default, shuffling is
+        #               NOT used for zfp but IS used with all other algorithms.
+        #
+        #       Available compression algorithms:
+        #
+        #           "zfp"
+        #               Use Peter Lindstrom's ZFP compression (
+        #               computation.llnl.gov/casc/zfp) Note: Whether this
+        #               compression is available is determined entirely at
+        #               run-time using the H5Z-ZFP compresser as a generic
```

Comment:
(style)  'compresser' may be misspelled - perhaps 'compressor'?

Reply:
Fixed